build/protos/kvstore.proto

```diff
@@ -29,6 +29,15 @@ message RemoveReply {
   // Empty because success/failure is signaled via GRPC status.
 }
 
+message KVMap {
+  repeated KVPair kvpairs = 1;
+}
```

Comment:
Added a key value map message for serialization and parsing to/from my data file. This allowed protobuf to deal with delimiting the data.

Reply:
This is good!